### DISCUSSION TITLE
Updating URLs of Services in a Web Map does not update the ItemID

### DISCUSSION THREAD

**Discussion by gthomasb** [EsriPS/arcgis-assistant-feedback#60] [General]
After updating the URLs of Services in a Web Map, when reviewing the map's json, the ItemID still references the original dataset.  Is there an easy way to correct this without having to update all the ItemID values by hand?

**Comment by lindak77** [comment]
We are having a similar issue.  We recently republished and overwrote one of our utility feature services published from 12 sanitary feature classes.  The numbers at the end of each URL changed (we're not sure why this happened as we didn't change the order of nor quantity of the feature classes in our Pro project before overwriting the feature service).  We were hoping to use AGO Assistant to quickly correct the numbers at the end of each URL in each of our web maps.  After making the changes with AGO Assistant, saving the changes and getting a "successful save" message, we can see that some of the URL's change, some do not and some change to a completely different number than we typed in when making the correction.  
![AGOAssistantScreenshot](https://user-images.githubusercontent.com/65732831/221897272-f2f20b99-abed-4f95-9fac-d7828e9f1cb4.png)
